aws/lambda/python/reports_notification.py

```
@@ -37,31 +40,53 @@
-    } 
-    response = client.publish(TopicArn=sns_topic_arn,MessageStructure='json',Message=json.dumps({'default': json.dumps(message_to_sns)}))
-    return response
+    }
+
+    try:
+        response = client.publish(
+            TopicArn=sns_topic_arn,
+            MessageStructure='json',
+            Message=json.dumps({'default': json.dumps(message_to_sns)}),
+        )
+        return response
```

Ruff

Consider moving this statement to an `else` block

(TRY300)